### ISSUE TITLE
Question regarding the learning rate update strategy

### ISSUE THREAD

**Issue by zhitao-guo** [fuying-wang/MGCA#40]
Thank you for your beautiful work~

I noticed that the learning rate is updated after every batch by default, which seems different from the more common strategies (update lr every epoch).
https://github.com/HKU-MedAI/MGCA/blob/3a2b4f25d47efc62e7d01d9204632a23d453afae/mgca/models/mgca/mgca_module.py#L405

 ```
scheduler = {
      "scheduler": lr_scheduler,
      "interval": "step",
      "frequency": 1
  }
```

Could you please explain the reason behind this approach? Additionally, if possible, could you provide some useful references or links on this topic? Looking forward to your reply~

**Comment by fuying-wang** [comment]
Hi,

We empirically use this step-level learning rate scheduler. Similar code snippet can be found in https://github.com/Lightning-Universe/lightning-bolts/blob/master/src/pl_bolts/models/self_supervised/simclr/simclr_module.py.

Thanks a lot!

**Comment by zhitao-guo** [comment]
Thanks very much.
